lib/v2/screens/receive_screens/receive_enter_data/receive_seeds_screen.dart

```diff
@@ -1,69 +1,107 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_bloc.dart';
 import 'package:seeds/v2/components/amount_entry/amount_entry_widget.dart';
 import 'package:seeds/v2/components/balance_row.dart';
 import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/v2/components/full_page_error_indicator.dart';
+import 'package:seeds/v2/components/full_page_loading_indicator.dart';
+import 'package:seeds/v2/components/snack_bar_info.dart';
 import 'package:seeds/v2/components/text_form_field_light.dart';
 import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'interactor/receive_enter_data_bloc.dart';
+import 'interactor/viewmodels/receive_enter_data_events.dart';
+import 'interactor/viewmodels/receive_enter_data_state.dart';
 
 class ReceiveEnterDataScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-        appBar: AppBar(title: const Text("Receive")),
-        body: Stack(
-          children: [
-            SingleChildScrollView(
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
-                  const SizedBox(height: 100),
-                  AmountEntryWidget(
-                    onValueChange: (value) {
-                      // BlocProvider.of<ReceiveEnterDataPageBloc>(context)
-                      //     .add(OnAmountChange(amountChanged: value));
-                    },
-                    autoFocus: true,
-                  ),
-                  const SizedBox(height: 36),
-                  Padding(
-                    padding: const EdgeInsets.symmetric(horizontal: horizontalEdgePadding),
-                    child: Column(
+    return BlocProvider(
+      create: (context) =>
+          ReceiveEnterDataBloc(BlocProvider.of<RatesBloc>(context).state)..add(const LoadUserBalance()),
+      child: Scaffold(
+          appBar: AppBar(title: const Text("Receive")),
+          body: BlocConsumer<ReceiveEnterDataBloc, ReceiveEnterDataState>(
+              listenWhen: (_, current) => current.pageCommand != null,
+              listener: (context, state) {
+                if (state.pageCommand is NavigateToReceiveDetails) {
+                  //Navigate to Receive Detail Page
+                }
+                if (state.pageCommand is ShowTransactionFailSnackBar) {
+                  ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
i think @RaulUrtecho  created a global snackbar we can use.

Reply:
And there it is, but it needs the method so that it is triggered just like the dialogues are triggered by a method.

Reply:
Not sure what you mean @RaulUrtecho 

@gguij004  we have a info snackbar that raul created. Please use it.

Reply:
my bad. ignore